.rustfmt.toml

```diff
@@ -0,0 +1,12 @@
+version = "Two"
```

Comment:
Would it be possible to split the format changes to a separate PR as well (i.e. leaving just the clippy config in this PR)? That would also allow me to play around with the settings a bit more, before deciding on what I would like to keep and what not.